README.md

```diff
@@ -1,6 +1,7 @@
 # ScenarioStateBehatExtension
 
 [![Build Status](https://travis-ci.org/gorghoa/ScenarioStateBehatExtension.svg?branch=master)](https://travis-ci.org/gorghoa/ScenarioStateBehatExtension)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/gorghoa/ScenarioStateBehatExtension/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/gorghoa/ScenarioStateBehatExtension/?branch=master)
```

Comment:
Waiting for #11 to be merged (we need to run phpunit in scrutinzer to generate code coverage)